### PR TITLE
removing gorm in services/job

### PR DESCRIPTION
**Pull request by pinebit**
Continuation of gorm removal for services/job

### CODE REVIEW

File: core/internal/cltest/factories.go [hunk 1]
```diff
@@ -732,7 +732,16 @@ func MustInsertExternalInitiatorWithOpts(t *testing.T, db *gorm.DB, opts Externa
 	ei.OutgoingToken = opts.OutgoingToken
 	token := auth.NewToken()
 	ei.AccessKey = token.AccessKey
-	err := db.Create(&ei).Error
+	ei.Salt = utils.NewSecret(utils.DefaultSecretSize)
+	hashedSecret, err := auth.HashedSecret(token, ei.Salt)
+	ei.HashedSecret = hashedSecret
+	require.NoError(t, err)
+	sql := `INSERT INTO external_initiators (created_at, updated_at, name, url, access_key, salt, hashed_secret, outgoing_secret, outgoing_token)
```
Comment (RensR) [inline core/internal/cltest/factories.go:739]: Could you extract a method for this to bridges/orm

**Review by RensR — CHANGES_REQUESTED** [review]

File: core/internal/cltest/factories.go (hunk 1 above)
Comment (pinebit) [reply core/internal/cltest/factories.go:739]: @RensR does it need to be done even for tests? (I don't know for sure, therefore double checking)

Comment (RensR) [reply core/internal/cltest/factories.go:739]: It seems it already exists CreateExternalInitiator(externalInitiator *ExternalInitiator) so you could use the existing one

Comment (pinebit) [reply core/internal/cltest/factories.go]: Done

**Review by RensR — APPROVED** [review]

**Review by RensR — APPROVED** [review]

**Review by samsondav — APPROVED** [review]

**Comment by samsondav** [comment]
@cl-homu r+

**Comment by cl-homu** [comment]
:pushpin: Commit 088e9cb629b140da9e74c88155fb5fb15fcc2cab has been approved by `samsondav`

**Comment by cl-homu** [comment]
:hourglass: Testing commit 088e9cb629b140da9e74c88155fb5fb15fcc2cab with merge 7628d5f0b6a2b58832a6ab1c6bbdcf7e8e145aae...

**Comment by cl-homu** [comment]
:sunny: Test successful - [checks-core](https://github.com/smartcontractkit/chainlink/runs/4211674035?check_suite_focus=true), [checks-lint](https://github.com/smartcontractkit/chainlink/runs/4211673839?check_suite_focus=true), [checks-npm](https://github.com/smartcontractkit/chainlink/runs/4211673891?check_suite_focus=true), [checks-operator](https://github.com/smartcontractkit/chainlink/runs/4211673842?check_suite_focus=true), [checks-prettier](https://github.com/smartcontractkit/chainlink/runs/4211674021?check_suite_focus=true), [checks-solidity](https://github.com/smartcontractkit/chainlink/runs/4211673985?check_suite_focus=true)
Approved by: samsondav
Pushing 7628d5f0b6a2b58832a6ab1c6bbdcf7e8e145aae to develop...

**Pull request merged**